src/java/io/bazel/rulesscala/test_discovery/DiscoveredTestSuite.scala

```diff
@@ -45,24 +45,30 @@ object PrefixSuffixTestDiscoveringSuite {
 
   private def discoverClasses(): Array[Class[_]] = {
 
-    val archive = archiveInputStream()
+    val archives = archivesPath.split(',')
+    val classes = archives.flatMap(discoverClassesIn)
+    if (classes.isEmpty)
+      throw new IllegalStateException("Was not able to discover any classes " +
+                                      s"for archive=$archives, " +
+                                      s"prefixes=$prefixes, " +
+                                      s"suffixes=$suffixes")
+    classes
+  }
+
+  private def discoverClassesIn(archivePath: String) = {
```

Comment:
can we add a return type here. I think it is `Array[Class[_]]`

Reply:
Will do

Reply:
done